### DISCUSSION TITLE
Changement le contenue de la page github après l'hebrgement du site sur github

### DISCUSSION THREAD

**Discussion by NesrineYAH** [community/community#40736] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

bonjour 
j'ai publié une page github directement dans mon espace personel gihub,  et j'ai oublié un mot, donc je voudrais le suprrimer sur ma page pour qu'il ne s'affichera pas au visiteurs, svp comment faire, j'ai besoin de votre solutions.
merci

**Comment by ossanna16** [comment]
Thanks for participating!
We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. What this means is that, until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
